modular_skyrat/modules/customization/__DEFINES/alt_sprites.dm

```diff
@@ -0,0 +1,2 @@
+#define HAS_TAUR_SPRITE 1
```

Comment:
opinion: these shouldn't be in the customization module, they should be in the skyrat defines folder